OpenEmu/pt.lproj/InfoPlist.strings

```diff
@@ -24,12 +24,12 @@
   SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-"Archived Game" = "Archived Game"; // TODO
+"Archived Game" = "Jogo arquivado";
 
-"GameCorePlugin" = "GameCorePlugin"; // TODO
+"GameCorePlugin" = "PluginDoNucleo"; // TODO NOT SURE HOW THIS SHOULD BE TRANSLATED, CONTEXT?
```

Comment:
Maybe "Plugin do core" or "Plugin do núcleo" I don't think we should translate `core` unless the core is translated in all other places.